src/core/createEvent.js

```diff
@@ -10,28 +10,92 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { clone, isEmptyObject, createMerger } from "../utils";
+import { isEmptyObject, deepAssign } from "../utils";
 
 export default () => {
   const content = {};
   let userXdm;
   let userData;
   let documentMayUnload = false;
-  let lastChanceCallback;
+  let isFinalized = false;
+  let shouldSendEvent = true;
+
+  const throwIfEventFinalized = methodName => {
+    if (isFinalized) {
+      throw new Error(
+        `${methodName} cannot be called after event is finalized`
+      );
+    }
+  };
 
   const event = {
+    getUserXdm() {
```

Comment:
It doesn't look like this is used anywhere.

Reply:
Nice catch, this was an artifact of working on the other `onLinkClick()` callback that we decided not to pursue